mixer/test/client/check_cache_hit/check_cache_hit_test.go

```diff
@@ -135,7 +135,7 @@ var reportAttributesOkGet = [...]string{`{
   "connection.mtls": false,
   "origin.ip": "[127 0 0 1]",
   "check.cache_hit": true,
-  "quota.cache_hit": true,
+  "quota.cache_hit": false,
```

Comment:
Can you explain why this flipped?

Reply:
When I added the new policy server counters to the mixer filter, I notice that quota.cache_hit is true even if there is no quota check.  So I fixed that.  cache_hit now means that you looked something up in a cache and actually found it.  If you skip the cache lookup, then cache hit is false.

Reply:
complements though on your eagle eye.  stuff like that should definitely be called out in a code review :)

Reply:
Sounds good. Thanks for explaining.